spec/models/user_access_spec.rb

```diff
@@ -6,41 +6,41 @@
   let(:power_user) { UserAccess.new(create(:admin, :power_user)) }
 
   describe "accessible_*" do
-    let!(:organization_1) { create(:organization) }
-    let!(:organization_2) { create(:organization) }
-    let!(:organization_3) { create(:organization) }
-
-    let!(:facility_group_1) { create(:facility_group, organization: organization_1) }
-    let!(:facility_group_2) { create(:facility_group, organization: organization_2) }
-    let!(:facility_group_3_1) { create(:facility_group, organization: organization_3) }
-    let!(:facility_group_3_2) { create(:facility_group, organization: organization_3) }
-
-    let!(:facility_1) { create(:facility, facility_group: facility_group_1) }
-    let!(:facility_2) { create(:facility, facility_group: facility_group_2) }
-    let!(:facility_3) { create(:facility, facility_group: facility_group_3_1) }
-    let!(:facility_4) { create(:facility, facility_group: facility_group_3_2) }
-    let!(:facility_5) { create(:facility) }
-    let!(:facility_6) { create(:facility) }
-
-    let!(:user_1) { create(:user, :with_phone_number_authentication, registration_facility: facility_1) }
-    let!(:user_2) { create(:user, :with_phone_number_authentication, registration_facility: facility_2) }
-    let!(:user_3) { create(:user, :with_phone_number_authentication, registration_facility: facility_3) }
-    let!(:user_4) { create(:user, :with_phone_number_authentication, registration_facility: facility_4) }
-    let!(:user_5) { create(:user, :with_phone_number_authentication, registration_facility: facility_5) }
-
-    let!(:admin_1) { create(:admin, :call_center, :with_access, resource: organization_1) }
-    let!(:admin_2) { create(:admin, :call_center, :with_access, resource: organization_1) }
-    let!(:admin_3) { create(:admin, :call_center, :with_access, resource: organization_3) }
-    let!(:admin_4) { create(:admin, :call_center, :with_access, resource: organization_3) }
-    let!(:admin_5) { create(:admin, :call_center) }
-
-    let!(:manager) { create(:admin, :manager) }
-    let!(:viewer_all) { create(:admin, :viewer_all) }
-    let!(:viewer_reports_only) { create(:admin, :viewer_reports_only) }
-    let!(:call_center) { create(:admin, :call_center) }
-
-    let!(:protocol_1) { create(:protocol) }
-    let!(:protocol_drug_1) { create(:protocol_drug) }
```

Comment:
`let!` is almost always not what we want, and makes tests harder to use and reason about...or in this case, impossible to turn on feature flags early enough to change how test objects are created.